packages/apps/docs/src/scripts/importReadme.mjs

```diff
@@ -0,0 +1,115 @@
+import * as fs from 'fs';
+import 'dotenv/config';
+import { remark } from 'remark';
+import { toMarkdown } from 'mdast-util-to-markdown';
+
+const DOCSROOT = './src/pages/docs/';
+
+const createFrontMatter = (title, menuTitle, order, editLink) => {
+  return `---
+title: ${title}  
+description: Kadena makes blockchain work for everyone.  
+menu: ${menuTitle}  
+label: ${title}  
+order: ${order}
+editLink: ${editLink}
+layout: full  
+---  
+`;
+};
+
+const createEditOverwrite = (filename) =>
+  `${process.env.NEXT_PUBLIC_GIT_EDIT_ROOT}/packages/${filename}`;
+
+const createGeneratedWarning = () => {
+  return `
+  
+  {/* 
+hier comes a comment 
+*/}
+  
+  `;
+};
+
+export const createSlug = (str) => {
+  if (!str) return '';
+  return str
+    .normalize('NFD')
+    .replace(/[\u0300-\u036f]/g, '')
+    .replace(/[\W_]+/g, '-')
+    .toLowerCase()
+    .replace(/^-+|-+$/g, '');
+};
+
+const getTitle = (pageAST) => {
+  const node = pageAST[0];
+  if (node.type !== 'heading' || node.depth !== 2) {
+    throw new Error('first node is not a Heading');
+  }
+
+  return node.children[0].value;
+};
+
+const createTreeRoot = (page) => ({
+  type: 'root',
+  children: page,
+});
+
+const createDir = (dir) => {
+  const dirArr = dir.replace(DOCSROOT, '').split('/');
+  dirArr.reduce((acc, val) => {
+    const newDir = `${acc}${val}/`;
+    console.log(newDir);
+    if (!fs.existsSync(newDir)) {
+      fs.mkdirSync(newDir);
```

Comment:
Perhaps you can use `recursive` to just create the dir (https://nodejs.org/api/fs.html#fsmkdirsyncpath-options)

Reply:
ughh...nice!